plugins/module_utils/aci.py

```diff
@@ -274,11 +278,15 @@ def cert_auth(self, path=None, payload='', method=None):
                                  'APIC-Certificate-DN=%s; ' % sig_dn +\
                                  'APIC-Certificate-Fingerprint=fingerprint; ' +\
                                  'APIC-Request-Signature=%s' % to_native(base64.b64encode(sig_signature))
+        self.get_auth = self.headers['Cookie']
```

Comment:
Why use get_auth when self.headers contains almost the same?

Reply:
Will take care of that